packages/account/src/Sections/Security/TwoFactorAuthentication/digit-form.tsx

```diff
@@ -77,12 +90,11 @@ const DigitForm = ({ is_enabled, setTwoFAStatus, setTwoFAChangedStatus, is_langu
                                     value={values.digit_code}
                                     onChange={e => {
                                         handleChange(e);
-                                        setReadyForVerification(false);
                                     }}
                                     onBlur={handleBlur}
                                     required
                                     error={touched.digit_code && errors.digit_code}
-                                    maxLength='6'
+                                    maxLength={6}
```

Comment:
just a doubt. are we following this format everywhere?

Reply:
Yes, in quite some places